game/unicode.cc

```diff
@@ -1,32 +1,34 @@
 #include "unicode.hh"
 
 #include "configuration.hh"
+#include "game.hh"
+
 #include <regex>
 #include <sstream>
 #include <stdexcept>
 #include <unicode/unistr.h>
 #include <unicode/ustream.h>
+#include <unicode/ubidi.h>
 #include "compact_enc_det/compact_enc_det.h"
 
-UErrorCode UnicodeUtil::m_staticIcuError = U_ZERO_ERROR;
+icu::ErrorCode UnicodeUtil::m_staticIcuError = icu::ErrorCode();
 icu::RuleBasedCollator UnicodeUtil::m_dummyCollator (icu::UnicodeString (""), icu::Collator::PRIMARY, m_staticIcuError);
 icu::RuleBasedCollator UnicodeUtil::m_sortCollator  (nullptr, icu::Collator::SECONDARY, m_staticIcuError);
```

Comment:
is that still usefull?

Reply:
What, the error? Yes, several ICU functions take a reference to an `UErrorCode` (or `icu::ErrorCode`)

Reply:
:m_sortCollator and friends ?

Reply:
Ah, yes. They are used when filtering songs.